seq2science/util.py

```diff
@@ -3,18 +3,25 @@
 """
 import colorsys
 import os
-from typing import List
-from math import ceil, floor
 import time
 import urllib.request
 import numpy as np
 import pandas as pd
 import pysradb
 import matplotlib.colors as mcolors
+
+from typing import List
+from math import ceil, floor
+from typing import List
+from snakemake.logging import logger
 from snakemake.io import expand
 from snakemake.exceptions import TerminatedException
```

Comment:
Just to be nit picky, the order is

import standardlib (e.g. os)
empty line
import packages (e.g. numpy pandas)
empty line
import local files